boards/FSM/Inc/App/configs/App_FlowRateThresholds.h

```diff
@@ -2,7 +2,7 @@
 
 // The datasheet (https://www.adafruit.com/product/828) for the flow meter
 // indicates a working flow rate between 1L/min to 30L/min
-#define PRIMARY_FLOW_METER_MIN_FLOW_RATE 1.0f
-#define PRIMARY_FLOW_METER_MAX_FLOW_RATE 30.0f
-#define SECONDARY_FLOW_METER_MIN_FLOW_RATE 1.0f
-#define SECONDARY_FLOW_METER_MAX_FLOW_RATE 30.0f
+#define PRIMARY_FLOW_METER_MIN_FLOW_RATE_L_PER_MIN 1.0f
```

Comment:
Unrelated to the PR but added units for the following `App_xThresholds.h`

Reply:
Fine with me this time but in the future would be nice to just do it in a separate PR

Reply:
:+1: